lib/publisher.rb

```diff
@@ -0,0 +1,31 @@
+# Publisher for publishing a policy live onto GOV.UK.
+#
+# This handles the various actions that must happen to put a policy live on the
+# GOV.UK website. This currently means:
+#
+#   * Pushing a content item to the Publishing API
+#   * Adding the policy to the search index
+#   * Re-publishing any parent policies to the publshing API to keep the
+#     "related" policies of the parents up-to-date.
+#
+class Publisher
+  attr_reader :policy
+
+  def initialize(policy)
+    @policy = policy
+  end
+
+  def publish!
```

Comment:
After discussion IRL, updated the comments on the class to indicated the purpose of this class and the tech debt it is introducing.